engine/access/rpc/backend/backend.go

```diff
@@ -123,6 +125,15 @@ func New(params Params) (*Backend, error) {
 		}
 	}
 
+	txErrorMessagesCacheSize := params.TxErrorMessagesCacheSize
+	if txErrorMessagesCacheSize == 0 {
+		txErrorMessagesCacheSize = 1000 // use a default value in case it wasn't configured
+	}
+	txErrorMessagesCache, err := lru.New[flow.Identifier, string](int(txErrorMessagesCacheSize))
```

Comment:
I think we should return an error in the node bootstrap if the size is 0 instead of silently changing it
```suggestion
	txErrorMessagesCache, err := lru.New[flow.Identifier, string](int(params.TxErrorMessagesCacheSize))
```